Add empty programs tab

hugo-vrijswijk opened the pull request:
The tab is now always shown. I'm not sure if this is what is needed, or based on what we should hide/show the tab. Input is welcome 😅.

Also adds a migration for the tab layout in the userPreferences db.


<img width="595" alt="image" src="https://github.com/gemini-hlsw/explore/assets/10114577/6d03e7bf-19cf-4070-9f85-18f5a4a0f1da">

cquiroz commented:
Can you test on phone sized screens, the tabs go to the bottom and may not fit now

hugo-vrijswijk commented:
@cquiroz It does become a bit cramped. Should we have the buttons overflow? 

<img width="400" alt="image" src="https://github.com/gemini-hlsw/explore/assets/10114577/521b0361-3e5c-4319-ac10-554e28f3a0ee">

hugo-vrijswijk commented:
I've updated it to have an overflow, so when the viewport is too small you can just scroll sideways. 

I've also fixed the 1px slide that the buttons do (on desktop too) when hovering over them. That's been bothering me since I started 😅

https://github.com/gemini-hlsw/explore/assets/10114577/275ff08f-d429-4424-851c-18e446cd274e
